README.md

```diff
@@ -1,24 +1,49 @@
+[![Travis CI Build Status](https://travis-ci.org/mozilla/rkv.svg?branch=master)](https://travis-ci.org/mozilla/rkv)
+[![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/lk936u5y5bi6qafb/branch/master?svg=true)](https://ci.appveyor.com/project/mykmelez/rkv/branch/master)
```

Comment:
Should this be pointing to "project/mkymelez"?  Can we get it under Mozilla's banner?

Reply:
Confusingly, the AppVeyor project at https://ci.appveyor.com/project/mykmelez/rkv builds the **mozilla/rkv** repo, while another project I created at https://ci.appveyor.com/project/mykmelez/rkv-iy5s5 builds the **mykmelez/rkv** repo.

That's because the "mykmelez" in the URL refers to the GitHub account name of the AppVeyor user who created the project, not the GitHub account/organization of the repository being built.

Interestingly, the github-owners mailing list is just discussing this issue, as @Mossop has an AppVeyor account with the username "Mozilla" that he can use to create projects with the URL prefix https://ci.appveyor.com/project/mozilla/. (I'm not actually sure how he obtained that username, as I can't find a way to change my own; it appears to be tied to my GitHub account name.)

In any case, that conversation seems to be leaning toward getting rid of that AppVeyor account in favor of individual Mozillians creating AppVeyor projects for their mozilla/* repos as desired, which is completely self-service and doesn't require central administration through Dave.

I'm open to counter-arguments, if you think that the URL issue will confound users enough to make it worth maintaining that central account.

Also note that it's technically possible to create a badge URL that refers to a GitHub repository rather than an AppVeyor project. Here's such a URL for mozilla/rkv:

https://ci.appveyor.com/api/projects/status/github/mozilla/rkv?svg=true&branch=master

The issue with that URL is that it'll stop working if someone else sets up a second project for the same repository, and literally any AppVeyor user can create a project for a public GitHub repository, whether or not they have any relationship to it. See https://github.com/appveyor/ci/issues/1525 for the gory details.

And even with such a badge URL, the project URL—i.e. the URL you visit when you click the badge—will still reflect the account name of the user who created it. Thus, for the project I created to build mozilla/rkv, the best we could do is:

`[![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/github/mozilla/rkv?svg=true&branch=master)](https://ci.appveyor.com/project/mykmelez/rkv/branch/master)`

Which doesn't seem worth it, given how easy it is for anyone to DOS the badge URL.